proxy/http2/Http2Stream.cc

```diff
@@ -163,6 +164,8 @@ Http2Stream::send_request(Http2ConnectionState &cstate)
       block = request_buffer.get_current_block();
     }
     done = _req_header.print(block->start(), block->write_avail(), &bufindex, &tmp);
+    TraceIn(h2_trace, cstate.ua_session->get_netvc()->get_remote_addr(), cstate.ua_session->get_netvc()->get_remote_port(),
```

Comment:
This outputs incoming HTTP/1.1 format request like "GET / HTTP/1.1" and "Host: example.com", right?
Since TraceIn is for wire tracing, I think it should output raw binary data or HTTP/2 format request like ":method: GET" and ":authority: example.com".